lua/wikis/commons/Widget/Match/Ticker/Container.lua

```diff
@@ -48,31 +44,27 @@ function MatchTickerContainer:render()
 
 	---@param type 'upcoming' | 'recent'
 	local function buildTemplateExpansionString(type)
-		local config = self.defaultProps.matchTicker
-
 		return String.interpolate(
 			'#invoke:Lua|invoke|module=${module}|fn=${fn}${args}',
 			{
-				module = config.module,
-				fn = config.fn,
-				args = table.concat(Array.map(
-					Table.entries(Table.merge(config.args, {type=type, dev=devFlag})),
-					function (entry)
-						return String.interpolate('|${key}=${value}', {key = entry[1], value = entry[2]})
+				module = self.defaultProps.module,
+				fn = self.defaultProps.fn,
```

Comment:
For some reasons i don't yet understand accessing these via `self.props.module`/`self.props.fn` does not work.
Same for these props in callTemplate, then there's even an error about `widgetClass` being nil